src/handwritten/system/storage/persisted_scale.py

```diff
@@ -17,12 +17,14 @@ class PersistedScale:
     """
     __slots__ = ['_name', '__weakref__']
 
-    def __init__(self, name):
+    def __init__(self, name, interpreter):
```

Comment:
This is a public constructor. Adding a required interpreter parameter breaks compatibility with client code that directly constructs it like `nidaqmx.system.storage.PersistedScale("MyScale")`.

See my comments on `scale.py.mako`.